src/main/java/edu/utah/ece/async/sboldesigner/sbol/editor/Part.java

```diff
@@ -122,16 +122,44 @@ public Image getImage() {
 		return smallImage;
 	}
 
+
 	/**
 	 * Creates a CD in design using roles.
 	 */
-	public ComponentDefinition createComponentDefinition(SBOLDocument design) {
+	public ComponentDefinition createComponentDefinition(SBOLDocument design) 
+	{
+		return createComponentDefinition(design, ComponentDefinition.DNA);
+	}
+	
+	/**
+	 * Create a ComponentDefinition that will set the roleType base on the given ComponentDefinition Type.

```

Comment:
the will set the roleType *based* on the given...

Reply:
Also, the field is probably more clearly named with "type" since that's what the field is called.